source/nl/1.0.0/index.html.haml

```diff
@@ -0,0 +1,299 @@
+
+  %p
+    GitHub Releases maken changelog wat alleen getoond kan worden aan gebruikers
+    binnen de context van GitHub. Het is mogelijk om deze dicht bij het format
+    te krijgen wat wij hier promoten, maar er zal iets meer werk voor nodig zijn.
+
+  %p
+    De huidige versie van GitHub releases is naar mijn mening niet
+    echt goed vindbaar voor gebruikers, in tegenstelling tot de typische
+    bestanden die in een naam in hoofdletters hebben
+    (<code>README</code>, <code>CONTRIBUTING</code>, etc.).
+    Een ander knelpunt is dat de interface geen links toestaat naar
+    commit logs van elke release.
+
+  %h4#automatic
+    %a.anchor{ href: "#automatic", aria_hidden: "true" }
+    Kunnen changelogs automatisch geparsed worden?
+
+  %p
+    Dat is lasig, mensen gebruiken immers veel verschillende formats en bestandsnamen.
+
+  %p
+    #{link_to "Vandamme", vandamme} is een Ruby gem van het
+    #{link_to "Gemnasium", gemnasium} team wat de changelogs van veel (maar niet alle)
+    open source projecten kan parsen.
+
+
+  %h4#yanked
+    %a.anchor{ href: "#yanked", aria_hidden: "true" }
+    Wat doen we met teruggetrokken (yanked) releases?
+
+  %p
+    Teruggetrokken releases zijn versies die teruggetrokken zijn als gevolg
+    van een serieuze bug of beveiligings probleem. Vaak zijn ze niet eens te zien in
+    de changelogs. Dat zou wel moeten. Zo zou je een teruggetrokken release moeten tonen:
+
+  %p <code>## 0.0.5 - 2014-12-13 [YANKED]</code>
+
+  %p
+    De <code>[YANKED]</code> tag is in hoofdletters voor een reden. Het is belanrijk
+    dat mensen dit zien. Omdat het tussen blokhaken genoteerd is, is het ook makkelijker
+    automatisch te parsen.
+
+
+  %h4#rewrite
+    %a.anchor{ href: "#rewrite", aria_hidden: "true" }
+    Mag je een changelog aanpassen/herschrijven?
+
+  %p
+    Natuurlijk. Er zijn goede redenen om een changelog te verbeteren.
+    Ik open regelmatig een pull request om missende releases toe te
+    voegen aan open source projecten met een slecht onderhouden changelog.
+
+  %p
+    Het kan ook zo zijn dat je ontdekt dat je een belanrijke aanpassing niet
+    vermeld hebt in je changelog. Het is dan natuurlijk zaak om dit alsnog
+    in je changelog te vermelden.
+
+  %h4#contribute
+    %a.anchor{ href: "#contribute", aria_hidden: "true" }
+    Hoe kan ik bijdragen?
+
+  %p
+    Dit document is niet de <strong>waarheid</strong>; het is mijn
+    weloverwogen mening, samen met wat informatie en voorbeelden die ik verzameld heb.
+
+  %p
+    Dit heb ik gedaan omdat ik wil dat de programmeer gemeenschap een consensus bereikt.
+    Ik denk dat de discussie net zo belanrijk is als het eindresultaat.
+
+  %p
+    Dus <strong>#{link_to "alle hulp is welkom", gh}</strong>.
+
+.press
+  %h3 Conversaties
+  %p
+    Ik was te gast bij #{link_to "The Changelog podcast", thechangelog} om te praten over
+    waarom een changelog belanrijk zou moeten zijn voor programmeurs, en over mijn motivatie achter dit project.
```

Comment:
suggestion: "...een changelog belangrijk zou moeten zijn voor programmeurs, ..." => "...een changelog belangrijk is voor programmers, ..."